zenoh/src/net/routing/dispatcher/queries.rs

```diff
@@ -156,7 +156,7 @@ pub(crate) fn undeclare_queryable(
                 }
             },
             None => {
-                tracing::error!(
+                tracing::trace!(
```

Comment:
This one should not occur with proper ACL (expo is not empty). Please revert to error.